examples/monolithic_build_multilevel_native/mlkem_native_all.c

```diff
@@ -10,8 +10,10 @@
 /* Include level-independent code */
 #define MLKEM_NATIVE_MULTILEVEL_BUILD_WITH_SHARED 1
 /* Include C files accompanying native code */
-#define MLKEM_NATIVE_MONOBUILD_WITH_NATIVE_ARITH
-#define MLKEM_NATIVE_MONOBUILD_WITH_NATIVE_FIPS202
+#define MLKEM_NATIVE_MONOBUILD_WITH_NATIVE_ARITH_AARCH64
+#define MLKEM_NATIVE_MONOBUILD_WITH_NATIVE_FIPS202_AARCH64
+#define MLKEM_NATIVE_MONOBUILD_WITH_NATIVE_ARITH_X86_64
+#define MLKEM_NATIVE_MONOBUILD_WITH_NATIVE_FIPS202_X86_64
```

Comment:
What's the reason for the refinement? Can't you achieve the same by guarding the 
```
#define MLKEM_NATIVE_MONOBUILD_WITH_NATIVE_ARITH
#define MLKEM_NATIVE_MONOBUILD_WITH_NATIVE_FIPS202
``` 
depending on the architecture you are building for?

Reply:
If you want a build of mlkem-native with, say, aarch64 backend but without x86_64 backend, the mechanism you propose does not work, and one would need to manually adjust the monobuild file.

Reply:
Mentioned that case in the commit message.

Reply:
Why can I not do
```
#if defined(__AARCH64EL__)
#define MLKEM_NATIVE_MONOBUILD_WITH_NATIVE_ARITH
#define MLKEM_NATIVE_MONOBUILD_WITH_NATIVE_FIPS202
#endif 
```
?

Reply:
Yes that works, thank you. Adjusted